model/registry/geo.yaml

```
@@ -0,0 +1,84 @@
+groups:
+  - id: registry.geo
+    type: attribute_group
+    prefix: geo
+    brief: >
+      Geo fields can carry data about a specific location related to an event.
+      This geolocation information can be derived from techniques such as Geo IP, or be user-supplied.
+
+      Note: Geo attributes are typically used under another namespace, such as client.* and describe the location of the corresponding
+      entity (device, end-user, etc). Semantic conventions that reference geo attributes (as a root namespace) or embed them
+      (under their own namespace) SHOULD document what geo attributes describe in the scope of that convention.
+
+    attributes:
+      - id: city.name
```

```suggestion
      - id: geo.city.name
```

(and in other places)